Automattic/wp-calypso
# Remove agencies survey banner

Related to #

## Proposed Changes

This PR removes the agencies survey banner that was added in https://github.com/Automattic/wp-calypso/pull/75353

There weren't many responses, so we decided to remove it

Conversation: p1HpG7-maT-p2#comment-62879

## Testing Instructions

1. Follow the testing instructions in https://github.com/Automattic/wp-calypso/pull/75353
2. Make sure the banner does not show up
![image](https://user-images.githubusercontent.com/65001528/236915346-99e5fd34-df95-4716-b0a9-e022064d85fa.png)

## Pre-merge Checklist
